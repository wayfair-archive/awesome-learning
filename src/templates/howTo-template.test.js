import React from 'react';
import { mount } from "enzyme";
import HowToTemplate from "./howTo-template";

describe("HowTo template", () => {
  it('matches the snapshot', () => {
    const wrapper = mount(<HowToTemplate />);
    expect(wrapper).toMatchSnapshot();
  });
});
