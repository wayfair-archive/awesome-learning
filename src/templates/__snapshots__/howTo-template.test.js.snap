// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowTo template matches the snapshot 1`] = `
ReactWrapper {
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "getWrappingComponentRenderer": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": Object {
      "instance": <h1>
        hey
      </h1>,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "hey",
      },
      "ref": null,
      "rendered": Array [
        "hey",
      ],
      "type": "h1",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": Object {
        "instance": <h1>
          hey
        </h1>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "hey",
        },
        "ref": null,
        "rendered": Array [
          "hey",
        ],
        "type": "h1",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__linkedRoots__): Array [],
  Symbol(enzyme.__unrendered__): <HowTo />,
  Symbol(enzyme.__updatedBy__): null,
}
`;
